mesh_msgs/PolygonMesh reference_mesh

string[] target_object_types
string target_object_geometry_type

bool fully_within
---
semap_msgs/ObjectPair[] pairs
